### PR TITLE
Add blank number format for whitespace-aware CTRL_A and CTRL_X

### PR DESCRIPTION
**Pull request by distobs**
Implement context-based incrementing and decrementing as suggested in #15033.

### CODE REVIEW

File: src/ops.c [hunk 1]
```diff
@@ -2813,8 +2816,15 @@ do_addsub(
 		&& (!has_mbyte || !(*mb_head_off)(ptr, ptr + col - 1))
 		&& !do_unsigned)
 	{
-	    negative = TRUE;
-	    was_positive = FALSE;
+	    if (do_blank && col >= 2 && !VIM_ISWHITE(ptr[col - 2]))
+	    {
```
Comment (chrisbra) [inline src/ops.c]: braces are not needed for single statements

File: src/ops.c [hunk 2]
```diff
@@ -2875,10 +2885,18 @@ do_addsub(
 		&& !visual
 		&& !do_unsigned)
 	{
-	    // negative number
-	    --col;
-	    negative = TRUE;
+	    if (do_blank && col >= 2 && !VIM_ISWHITE(ptr[col - 2]))
+	    {
```
Comment (chrisbra) [inline src/ops.c]: braces are not needed for single statements

**Comment by chrisbra** [comment]
thanks, that looks useful!

**Comment by distobs** [comment]
Done!

**Comment by chrisbra** [comment]
thanks!

**Pull request closed without merging**